packages/fmg-core/src/state.js

```diff
@@ -28,6 +28,14 @@ class State {
 
     return [ r, s, v ];
   }
+
+  get numberOfParticipants() {
+    return this.channel.numberOfParticipants;
+  }
+
+  get mover() {
```

Comment:
💡  `s/participant/player/`?

Reply:
This is currently consistent with https://github.com/magmo/force-move-games/blob/master/packages/fmg-core/contracts/State.sol#L33, but I'm open to changing them all.